### DISCUSSION TITLE
Negative aerosol surface flux over ice?

### DISCUSSION THREAD

**Discussion by AlejandroBaroPerez** [ARM-Development/comble-mip#77] [General]
In a preliminary simulation of COMBLE Part 2 (in which I am using the wind and SST dependent particle flux) I notice that, before the ice melts, there is a loss of aerosol near the surface (the aerosol surface flux is negative). Once the ice melts the aerosol surface flux is positive, leading to a visible increase in the aerosol concentrations near the surface.  Although it doesn't seem strange to me I still want to ask if we are expecting to have a near-surface aerosol loss over the ice at the beginning of the simulation?

**Comment by jchylik** [comment]
> loss of aerosol near the surface (the aerosol surface flux is negative)

This sounds like a mistake in the surface parametrization in the model you are using.
I was also getting negative values at first, but then I set
`S_{srf} = max ( 0.0,  formula[Jaegle et al. (2011)]  ) `

https://arm-development.github.io/comble-mip/notebooks/setup/aerosol-surface.html#jaegle-et-al-2011-surface-particle-flux-integrated-over-all-sizes

**Reply by AlejandroBaroPerez** [reply]
Thanks for the feedback! Actually I checked that, for SST < -3ºC,  variable Fn (particle number per squared centimeter and second) becomes negative. So this doesn't look like an error (unless there something else I am forgetting...) . But I can apply your strategy as well if we are not supposed to have negative surface fluxes values. That's why I wanted to hear from @twjuliano or @fridlind about this.  Thanks

**Reply by jchylik** [reply]
> Actually I checked that, for SST < -3ºC, variable Fn becomes negative.

The salty water freezes already by -2ºC, so then it does not produce sea-spray. 

And if we want to be pedantic, the parametrization of Jaeglé[^1] is based on empirical data between 0ºC and 30ºC.
See **Figure 6**[^1].  
So what we do on the interval [-3ºC,0ºC] is purely extrapolation.

[^1]: Jaeglé, L., Quinn, P. K., Bates, T. S., Alexander, B., and Lin, J.-T.: Global distribution of sea salt aerosols: new constraints from in situ and remote sensing observations, Atmos. Chem. Phys., 11, 3137–3157, https://doi.org/10.5194/acp-11-3137-2011, 2011.

**Reply by floriantornow** [reply]
Hi Alejandro and Jan,

Alejandro, thank you for raising this issue and apologies for the lack of clarity. Please use Jan's setting that essentially treats early surfaces (i.e., SST < 0 ºC) as non-emitting. 

Jan, thank you for quickly providing a solution. Assuming you don't mind, I'll update the Github page using your formulation. 

Thank you,
Florian

**Reply by jchylik** [reply]
> early surfaces (i.e., SST < 0 ºC) as non-emitting

It is still a question where to put the lower bound:

-   0.0 ºC : because of the fitted line in the paper starts there
- -0.2 ºC : because of the data in the paper start there
- -2 ºC : freezing point of salty water

**Comment by floriantornow** [comment]
Hi Alejandro and Jan,

Following up on your very good question, we performed a set of sensitivity runs to test the downwind impact. The attached plot shows two LES runs with the following configurations: (1) allowing sea-spray fluxes at all sea surface temperatures (SST) and setting SST to zero for SST < 0 °C, (2) only allowing sea-spray fluxes at SST > 0 °C (and setting fluxes to zero for SST < 0 °C).

Apart from a ~20 % greater peak CDNC at 3-4 hours (panel g) for configuration (1), all other variables suggest no difference to the overall cloud field. Most deviations occur after 10 hours and appear within the expected noise range that an ensemble with the same configuration would show or other uncertainties would amount to (e.g., [from an alternative surface treatment](https://arm-development.github.io/comble-mip/notebooks/plotting/example_plotting_specification_tests.html)). 

With an eye toward climate model implementations, the simpler approach would be configuration (1), which does not require modelers to prepare case-specific code. Given the lack of impact and the preferred simplicity of one configuration over the other, we ask participants to include an aerosol surface flux from all SST and allow surface fluxes from supercooled sea surfaces, using 0 °C flux values at all supercooled temperatures. Please find the [GitHub page](https://arm-development.github.io/comble-mip/notebooks/setup/aerosol-surface.html) updated accordingly.

Thank you and sorry for any inconvenience,
Florian

![comble_SSAflux_cloudprop_filtered_1](https://github.com/ARM-Development/comble-mip/assets/133912814/6bf19571-9482-4e2c-b3c0-be060019a958)
